content/microcontrollers/esp8266-nodemcu.md

```diff
@@ -60,7 +60,7 @@ Now you should be able to flash your board as follows:
 - Build and flash your TinyGo code using the `tinygo flash` command. This command flashes the ESP8266 with the blinky1 example:
 
     ```
-    tinygo flash -target=esp8266 examples/blinky1
+    tinygo flash -target=nodemcu examples/blinky1
```

Comment:
```suggestion
    tinygo flash -target=nodemcu -port=/dev/ttyUSB examples/blinky1
```

Reply:
I think the esptool can locate the port itself on macOS. If not, let's handle in subsequent PR so we can get everything else done.